docs/api-markdown-documenter/render-api-documentation.js

```diff
@@ -87,11 +87,14 @@ export async function renderApiDocumentation(inputDir, outputDir, uriRootDir, ap
 				alerts.push("Deprecated");
 			}
 
+			// TODO: Use `getModifierTag` helper once added to `@fluid-tools/api-markdown-documenter`
+			if (apiItem.tsdocComment?.modifierTagSet?.hasTagName("@legacy")) {
+				alerts.push("Legacy");
+			}
+
 			const releaseTag = ApiItemUtilities.getReleaseTag(apiItem);
 			if (releaseTag === ReleaseTag.Alpha) {
-				// Temporary workaround for the current `@alpha` => "Legacy" state.
-				// This should be replaced with "Alpha" once that has been cleaned up.
-				alerts.push("Legacy");
+				alerts.push("Alpha");
 			} else if (releaseTag === ReleaseTag.Beta) {
 				alerts.push("Beta");
 			}
```

Comment:
Isn't this going to accumulate multiple alerts? If Legacy, then I don't think we want the others... at least not as they stand today.

Reply:
Since the support contract is separate from the "legacy"-ness, including (potentially) both "Legacy" and the sub-public release tag made sense to me. But this is flexible. I would defer to you here.

Reply:
I've good with it. I didn't know how to translate `alerts.push("Alpha")` into website results, but now I think it is just some more text in a list. That sounds good to me.